### DISCUSSION TITLE
How to add classes (and other attributes to dynamically constructed form elements with FormKit Schema?

### DISCUSSION THREAD

**Discussion by nielsbom** [formkit/formkit#1440] [General]
I have a FormKit schema I dynamically generate. A bunch of checkboxes for example.

I now want to change (append, reset, remove) some classes to the input elements that are created. For non-form elements I can use `attr: { class: ["warning"]}}` but that does not work for form elements. How should I change the classes of those FormKit schema form elements?

Thanks!

**Comment by justin-schroeder** [comment]
You just need to target the "section" inside the formkit input you want to target like `sectionName-class` (each input’s documentation page has a list of the available sections).  For example:

```js
{
  $formkit: 'text',
  outerClass: 'my classses go here'
}
```
